srcpkgs/nextdns/files/nextdns/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec /usr/bin/nextdns run ${OPTS}
```

Comment:
We usually use substitutions instead of shipping default service conf files.
```
exec /usr/bin/nextdns run ${OPTS:--config-file /etc/nextdns.conf}
```